pkg/cmd/admin/migrate/migrator.go

```diff
@@ -502,3 +502,20 @@ func DefaultRetriable(info *resource.Info, err error) error {
 	}
 	return err
 }
+
+// isNotFoundForInfo returns true iff the error is a not found for the the specific info object.
+func isNotFoundForInfo(info *resource.Info, err error) bool {
+	if err == nil || !errors.IsNotFound(err) {
+		return false
+	}
+	status, ok := err.(errors.APIStatus)
+	if !ok {
+		return false
+	}
+	details := status.Status().Details
+	if details == nil {
+		return false
+	}
+	gvk := info.Object.GetObjectKind().GroupVersionKind()
```

Comment:
does this work correctly with not found errors for legacy group and new API groups?